src/content/changelog/4-138-0-2022-10-17.md

```diff
@@ -0,0 +1,29 @@
+---
+title: '4.138.0: 2022-10-17'
+date: 2022-10-17T01:00:00.000Z
+version: 4.138.0
+changelog:
+  - API
+---
+
+### Changed
+
+* **Domain Record Create** ([POST /domains/{domainId}/records](https://www.linode.com/docs/api/domains/#domain-record-create))
```

Comment:
Should there be a description below this? Or are the update notes below valid for this line also?